### PR TITLE
Divider: token updates for strong and subtle variants

### LINKED ISSUE #26326: [Bug]: Divider color token assignments for content and icon in subtle and strong variants
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
React V9

https://aka.ms/fluentui-storybook
```

### Are you reporting Accessibility issue?

no

### Reproduction

React Components / v9 - https://codesandbox.io/

### Bug Description

Design oversight, design spec did not match what was drawn in the example

## Actual Behavior
Currently in subtle variant, content uses Neutral Foreground 2, whereas icon uses Neutral foreground 3
Currently in strong variant, content uses Neutral Foreground 2, whereas icon uses Neutral foreground 1 

## Expected Behavior
In Subtle variant, Neutral Foreground 3 for both content and icon
image
https://www.figma.com/file/TQO8qAOcrwU3ig2CTcan0e/Divider?node-id=621%3A3457&t=eKMxFz3zUNnqWPjP-4

In Strong variant, Neutral foreground 1 for both content and icon
image
https://www.figma.com/file/TQO8qAOcrwU3ig2CTcan0e/Divider?node-id=621%3A3730&t=eKMxFz3zUNnqWPjP-4

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by kkakroo**
## Previous Behavior
Subtle variant had color of colorNeutralForeground2
Strong variant had color of colorNeutralForeground2

## New Behavior
Subtle variant now has color of colorNeutralForeground3
Strong variant now has color of colorNeutralForeground1

## Related Issue(s)

- Fixes #26326

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                             |   Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                  Change |
| :-------------------------------------------------------------------------------------------- | -------------------------: | --------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr> | `19.754 kB`<br />`6.68 kB` | `19.856 kB`<br />`6.707 kB` | `102 B` <br />`27 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |   `67.03 kB`<br />`18.099 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `206.019 kB`<br />`57.065 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `37.861 kB`<br />`12.034 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.324 kB`<br />`2.129 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4fffed69a86831eb04a3a9adff78fdb266f4c3a0'>4fffed69a86831eb04a3a9adff78fdb266f4c3a0</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Avatar-mount_base.html">1307</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Avatar-mount.html">1287</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Button-mount_base.html">917</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Button-mount.html">945</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Field-mount_base.html">1957</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Field-mount.html">1935</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProvider-mount_base.html">1557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProvider-mount.html">1565</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProviderWithTheme-mount_base.html">575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProviderWithTheme-mount.html">590</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">536</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">537</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">580</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">578</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/InfoButton-mount_base.html">520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/InfoButton-mount.html">523</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/MakeStyles-mount_base.html">1955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/MakeStyles-mount.html">1955</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Persona-mount_base.html">2901</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/Persona-mount.html">2828</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/SpinButton-mount_base.html">2285</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27169/perf-test-react-components/SpinButton-mount.html">2259</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-divider-0e328352-ef9a-449b-aa18-b4f40cbd9e75.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-divider-0e328352-ef9a-449b-aa18-b4f40cbd9e75.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 12 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27169&commitId=534b4c3fd60db772ba80ea21dce5554cfacbc346&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1880 screenshots unchanged, 0 screenshots with different dimensions and 12 screenshots with visible difference.


  <details><summary><b>unknown</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider Converged - Horizontal.Appearance strong - Dark Mode.default.chromium.png|14| Changed|
|Divider Converged - Horizontal.Appearance strong.default.chromium.png|10| Changed|
|Divider Converged - Horizontal.Appearance subtle - Dark Mode.default.chromium.png|14| Changed|
|Divider Converged - Horizontal.Appearance subtle.default.chromium.png|10| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|1831483| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|1832333| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|1827930| Changed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|1827931| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|1828108| Changed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|1828114| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|1827814| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|1821426| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

**Comment by kkakroo** [comment]
Hi @tomi-msft, can you please have a look at this PR when you have some time.

**Review by khmakoto — APPROVED** [review]

File: change/@fluentui-react-divider-0e328352-ef9a-449b-aa18-b4f40cbd9e75.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-divider-0e328352-ef9a-449b-aa18-b4f40cbd9e75.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=27169&commitId=6b9823d76b10da3200d299d769d7fa6f88257aa0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Divider Converged - Horizontal</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider Converged - Horizontal.Appearance strong.default.chromium.png|10| Changed|
|Divider Converged - Horizontal.Appearance subtle.default.chromium.png|10| Changed|
|Divider Converged - Horizontal.Appearance strong - Dark Mode.default.chromium.png|14| Changed|
|Divider Converged - Horizontal.Appearance subtle - Dark Mode.default.chromium.png|14| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Comment (khmakoto) [reply change/@fluentui-react-divider-0e328352-ef9a-449b-aa18-b4f40cbd9e75.json:1]: Changes are expected.

**Pull request merged**